src/korean.js

```diff
@@ -129,10 +128,18 @@ class Korean {
 
   doFuture (word) {
     const stem = breakdown(word.slice(0, -1));
-    if (stem[-1] !== 8) {
+    if (stem.length < 3) {
       stem.push(8);
+      return `${combineSymbols(stem)} 꺼야`;
```

Comment:
@minsooshin I think you're right again :) ,I looked it up. Nice catch!
I'll add another commit to fix this.